src/Api/Controllers/CiphersController.cs

```diff
@@ -387,17 +405,33 @@ public async Task PutDeleteAdmin(string id)
         }
 
         [HttpPut("delete")]
-        public async Task PutDeleteMany([FromBody]CipherBulkRestoreRequestModel model)
+        public async Task PutDeleteMany([FromBody]CipherBulkDeleteRequestModel model)
         {
             if (!_globalSettings.SelfHosted && model.Ids.Count() > 500)
             {
-                throw new BadRequestException("You can only restore up to 500 items at a time.");
+                throw new BadRequestException("You can only delete up to 500 items at a time.");
             }
 
             var userId = _userService.GetProperUserId(User).Value;
             await _cipherService.SoftDeleteManyAsync(model.Ids.Select(i => new Guid(i)), userId);
         }
```

Comment:
Yeah, Chad mentioned this one above too. It seems like there might have just been a long standing copy/paste issue here that never broke because the DeleteRequest and RestoreRequest models were identical. The api call from jslib sends a delete model to this method, and the action is delete.